libs/forms/src/lib/components/form-generator/form-generator.component.ts

```diff
@@ -69,14 +71,27 @@ export class FormGeneratorComponent implements OnInit {
   /** Datafields for dynamic data */
   @Input() datafields?: FormsLib.Datafields | null = {};
   /** Disable submit button. Otherwise will rely on the form validators to allow submission */
-  @Input() disableSubmit = false;
+  @Input() disableSubmit: null | boolean = false;
```

Comment:
did you want to do signals here?

Reply:
That is a separate lib that needs a complete overhaul, doing that right after this PR